blackbox/test_dns_discovery.py

```diff
@@ -84,3 +105,9 @@ def test_nodes_discover_each_other(self):
             c.execute('''select count(*) from sys.nodes''')
             result = c.fetchone()
         self.assertEqual(result[0], self.num_nodes, 'Nodes must be able to join')
+
+
+suite = TestSuite()
+suite.addTest(ParametrizedTestCase.parametrize(DnsSrvDiscoveryTest, useTcp=False))
```

Comment:
Could we do this instead with a sub-class - something like:


```
class BaseTestCase(TestCase):

    def setupDns(self, useTcp):
        ...



class TcpDnsDiscoTest(BaseTestCase):

    def setUp(self):
        self.setupDns(true)

...
```


The parameterized test case makes it a bit difficult to choose a single variant to run from within IDE/editor.

(Also fine to keep it - not a blocker9